sdk/storage/storage-blob/src/Clients.ts

```diff
@@ -3946,6 +3952,10 @@ export class BlockBlobClient extends BlobClient {
     const { span, spanOptions } = createSpan("BlockBlobClient-query", options.tracingOptions);
 
     try {
+      if (!isNode) {
+        throw new Error("This operation currently is only supported in Node.js.");
+      }
```

Comment:
Updated JSDoc to highlight this method Node.js only. Refer to other Node.js only or browser only method JSDoc.

Reply:
Added 'ONLY AVAILABLE IN NODE.JS RUNTIME.' in comments.